vizro-core/docs/pages/development/contributing.md

```diff
@@ -145,6 +145,21 @@ Vizro's version is given by `__version__` in [`src/vizro/__init__.py`](https://g
 
 To build the source distribution and wheel, run `hatch build`.
 
+### Steps to create a Vizro release
+
+The following steps are required to create a release of Vizro. Note that this is only done by maintainers:
+
+1. Checkout a new branch with a suitable name, e.g. `git checkout -b release/vx.y.z` (note that the branch name does not influence the versioning!)
```

Comment:
So the branch name would be matched with what's in the `__init__.py` right? Like currently it's `"0.1.1.dev0"` so the branch name should be release/v0.1.1

Shall we include the package name into the branch name as well? something like `release/vizro-core/v0.1.1`